astronomy/trappist_gravity_model.proto.txt

```diff
@@ -1,117 +1,120 @@
 gravity_model {
-  solar_system_frame : SKY
-  # Right ascensions from a normal distribution with μ=-90, σ=0.05.
-  # Declinations from a normal distribution with μ=0, σ=0.05.
+  solar_system_frame    : SKY
+  # Right ascensions from a normal distribution with μ=-90, σ=0.05°.
+  # Declinations from a normal distribution with μ=0, σ=0.05°.
+  # This standard deviation is the mode of the polar angle scatter distribution
+  # in figure 7 of Luger, Lustig-Yaeger, and Agol (2017), Planet-Planet
+  # Occultations in TRAPPIST-1 and Other Exoplanet Systems.
 
   # Star.
   # Mass and radius from "Early 2017 observations of TRAPPIST-1 with Spitzer",
   # Delrez et al., 2018.
   # Angular frequency from "Frequent flaring in the TRAPPIST-1 system — unsuited
   # for life?", Vida et al., 2017.
   body {
-    name                 : "Trappist-1"
-    mass                 : "0.0890 M☉"
-    reference_instant    : "JD2457000.000000000"
-    mean_radius          : "0.1210 R☉"
-    axis_right_ascension : "-89.9408 deg"
-    axis_declination     : "0.0497342 deg"
-    reference_angle      : "0 deg"
-    angular_frequency    : "109.256 deg / d"
+    name                    : "Trappist-1"
+    gravitational_parameter : "0.0890 GM☉"
```

Comment:
Note that this pull request does *not* regenerate the `cfg` files, as the tools would copy over `0.0890 GM☉` and the like instead of converting to SI units, which would cause interchange issues for strings passed in structs.

We want to rerun the optimization soon anyway.